apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -138,15 +145,33 @@ export function InstructorQuestionSettings({
                     slashes to separate directories.
                   </small>
                 </div>
-
-                <div class="table-responsive card mb-3">
+                <div class="table-responsive card mb-3 overflow-visible">
                   <table
                     class="table two-column-description"
                     aria-label="Question topic, tags, and assessments"
                   >
                     <tr>
-                      <th class="border-top-0">Topic</th>
-                      <td class="border-top-0">${TopicBadge(resLocals.topic)}</td>
+                      <th class="align-middle">
+                        <div>Topic</div>
+                      </th>
+                      <td>
+                        ${canEdit
+                          ? html` <select
+                              id="topic"
+                              name="topic"
+                              data-selected-topic="${resLocals.topic.name}"
+                            >
+                              ${courseTopics.map((topic) => {
+                                return html` <option
+                                  value="${topic.name}"
+                                  data-color="${topic.color}"
+                                  data-name="${topic.name}"
+                                  data-description="${topic.description}"
+                                ></option>`;
+                              })}
+                            </select>`
```

Comment:
Nit (plus or minus Prettier):

```suggestion
                          ? html`
                            <select
                              id="topic"
                              name="topic"
                              data-selected-topic="${resLocals.topic.name}"
                            >
                              ${courseTopics.map((topic) => {
                                return html`
                                  <option
                                  value="${topic.name}"
                                  data-color="${topic.color}"
                                  data-name="${topic.name}"
                                  data-description="${topic.description}"
                                ></option>
                                `;
                              })}
                            </select>
                          `
```

Reply:
Updated locally in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).